PKGBUILD

```diff
@@ -0,0 +1,56 @@
+# Maintainer: Jonas Renk <jonas.renk@stud.tu-darmstadt.de>
+pkgname=systemverilog
+pkgver=20210116
+pkgrel=1
+epoch=
+pkgdesc="Synthese und Simulation"
+arch=('any')
+url="https://moodle.informatik.tu-darmstadt.de/mod/folder/view.php?id=31324"
+license=('MIT')
+groups=()
+depends=('yosys'
+         'graphviz'
+         'iverilog'
+         'gtkwave')
+makedepends=('dos2unix'
+             'unzip')
+checkdepends=()
+optdepends=()
+provides=()
+conflicts=()
+replaces=()
+backup=()
+options=()
+install=
+changelog=
+source=("$pkgname-$pkgver.zip")
+noextract=("$pkgname-$pkgver.zip")
+md5sums=('3658f7ab86ba995e612178de6920825c')
```

Comment:
The date of change of the folders in the archive file is set by Moodle to the time of the download, so the are never two downloaded files with the same checksum. Are there any solutions for this problem?

Reply:
Only to `SKIP` validation, sadly.